src/config.rs

```diff
@@ -382,8 +385,12 @@ impl Config {
             RawConfig::new()
         };
 
-        // Convert to config
-        let mut config = Self::from_raw(raw_config).context("Could not process raw config")?;
+        // Safe to unwrap since we managed to read the file
+        let config_file_dir = config_file_path.parent().unwrap();
```

Comment:
The comment is still a bit off here, we are not always reading the file. The proper argument is that we know that this path points to a file, which necessarily has a directory, as you pointed out in the other thread. This all builds on the implementation detail of `get_config_path`. If we would inline `get_config_path`, then we would not need the `.parent` here at all, so maybe we should just do that? I suppose we could get rid of `get_config_path` altogether and also replace the second occurrence with a combination of `get_config_dir` and `CONFIG_FILE_NAME`.

Reply:
Personally would not inline the method, since that gives you the freedom to e.g. move the config file into a subdirectory. Agree that reconstructing the parent is not very pretty, imo it's more explicit than inlining the method ("we're resolving paths relative to the parent directory of the config file, whichever it is").

Reply:
Updated the comment.